src/main/java/com/github/herrschwarz/status4spring/cache/GuavaCacheStatsProvider.java

```diff
@@ -0,0 +1,46 @@
+package com.github.herrschwarz.status4spring.cache;
+
+import com.google.common.cache.Cache;
+import org.springframework.cache.guava.GuavaCacheManager;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+
+public class GuavaCacheStatsProvider implements CacheStatsProvider {
+
+    private GuavaCacheManager cacheManager;
+
+    public GuavaCacheStatsProvider(GuavaCacheManager cacheManager) {
+        this.cacheManager = cacheManager;
+    }
+
+
+    // @TODO: trait?
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Comment matches to-do format 'TODO:'.](https://www.codacy.com/app/HerrSchwarz/status4spring/file/2642488994/issues/source?bid=3309767&fileBranchId=3321843#l19)